tests/unit/spec/base/core-datalayer.js

```diff
@@ -67,6 +67,23 @@ describe('core-datalayer.js', function() {
         });
     });
 
+    describe('getLatestFxVersion', function() {
+
+        afterEach(function() {
+            $('html').removeAttr('data-latest-firefox');
+        });
+
+        it('will return the Firefox version from the data-latest-firefox attribute from the html element if present', function() {
+            $('html').attr('data-latest-firefox', '48.0');
+
+            expect(Mozilla.Analytics.getLatestFxVersion()).toBe('48.0');
+        });
+
+        it('will return null if no data-latest-firefox attribute is present on the html element', function() {
```

Comment:
Test says it should return `null` here but it's actually returning `undefined`? Not sure which one you're wanting, but though I should flag it.